inst/tinytest/test_sha1.R

```diff
@@ -467,3 +468,19 @@ for (algo in c("md5", "sha1", "crc32", "sha256", "sha512", "xxhash32",
     )
 
 }
+
```

Comment:
unit test for `sha.default()` with `is.list(x) == TRUE`
```suggestion
expect_true(is.character(sha1(sessionInfo())))
```